src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/wwwroot/css/site.css

```diff
@@ -8,6 +8,15 @@ a, .btn-link {
     color: #0366d6;
 }
 
+::-moz-selection {
+    color: #fff;
+    background-color: #333;
+}
+::selection {
+    color: #fff;
+    background-color: #333;
```

Comment:
Would this weird out if you have dark mode enabled in the browser?

Reply:
So I switched to dark and it looks ok since the colors didn't change, but everything else did around it including the contrast tool

![image](https://user-images.githubusercontent.com/6537861/124675158-f9961c80-de70-11eb-9097-cffe341f1a88.png)